Modules/Modules/Using Nested Paths to Clean Up Large use Lists/task.md

```diff
@@ -0,0 +1,39 @@
+### Using Nested Paths to Clean Up Large use Lists
+
+If we’re using multiple items defined in the same crate or same module, listing each item on its own line can take up a lot of vertical space in our files. For example, these two `use` statements bring items from `std` into scope:
+
+```rust
+    use std::cmp::Ordering;
+    use std::io;
+    // ---snip---
+```
+
+Instead, we can use nested paths to bring the same items into scope in one line. We do this by specifying the common part of the path, followed by two colons, and then curly brackets around a list of the parts of the paths that differ, as shown below:
+
+```rust
+    use std::{cmp::Ordering, io};
+    // ---snip---
+```
+
+##### Specifying a nested path to bring multiple items with the same prefix into scope
+
+In bigger programs, bringing many items into scope from the same crate or module using nested paths can reduce the number of separate `use` statements needed by a lot!
```

Comment:
In bigger programs, bringing many items into scope from the same crate or module using nested paths can significantly reduce the number of separate needed `use` statements!